subprojects/gradle/slack/src/main/kotlin/com/avito/slack/SlackClient.kt

```diff
@@ -5,29 +5,29 @@ package com.avito.slack
 
 import com.avito.android.Result
 import com.avito.slack.model.FoundMessage
-import com.avito.slack.model.SlackChannel
+import com.avito.slack.model.SlackChannelId
 import com.avito.slack.model.SlackMessage
 import com.avito.slack.model.SlackSendMessageRequest
-import com.avito.slack.model.strippedName
 import com.github.seratch.jslack.Slack
 import com.github.seratch.jslack.api.methods.MethodsClient
 import com.github.seratch.jslack.api.methods.SlackApiResponse
-import com.github.seratch.jslack.api.methods.request.channels.ChannelsHistoryRequest
-import com.github.seratch.jslack.api.methods.request.channels.ChannelsListRequest
 import com.github.seratch.jslack.api.methods.request.chat.ChatPostMessageRequest
 import com.github.seratch.jslack.api.methods.request.chat.ChatUpdateRequest
-import com.github.seratch.jslack.api.model.Channel
+import com.github.seratch.jslack.api.methods.request.conversations.ConversationsHistoryRequest
 import java.io.File
 
 interface SlackClient : SlackMessageSender, SlackFileUploader {
 
     fun updateMessage(
-        channel: SlackChannel,
+        channelId: SlackChannelId,
         text: String,
         messageTimestamp: String
     ): Result<SlackMessage>
 
-    fun findMessage(channel: SlackChannel, predicate: SlackMessageUpdateCondition): Result<FoundMessage>
+    fun findMessage(
+        channelId: SlackChannelId,
+        predicate: SlackMessageUpdateCondition
```

Comment:
I didn't get API
The method is named `findMessage` but updates a message.
I'd like to make API more obvious:
- Name it update...
- Or move responsibility for an update to client